api/src/main/java/edu/wpi/first/shuffleboard/api/sources/NetworkTableSource.java

```diff
@@ -1,8 +1,8 @@
-package edu.wpi.first.shuffleboard.sources;
+package edu.wpi.first.shuffleboard.api.sources;
```

Comment:
I mentioned in #101 that `SourceType` should be split into an interface and util class, like we have for `DataType`. The interface and util class would live in `:api`, and the implementations would be in `:app`. None/Static should still be defined in the util class IMO